laborer.js

```diff
@@ -206,43 +206,24 @@ exports.taskServerTypeScript = function(opt) {
 };
 
 
-var mochaParams = {
-  reporter: 'spec'
-};
-
-exports.taskUtilsTest = function() {
+var generateTester = function(path, parameters) {
   return function() {
-    return gulp.src('./build/utils/**/*.mocha.js', {read: false})
+    return gulp.src(path, {read: false})
       // gulp-mocha needs filepaths so you can't have any plugins before it
-      .pipe($.mocha(mochaParams));
+      .pipe($.mocha(parameters));
```

Comment:
This is cool, I think it would be nice to default the parameters to `{ reporter: 'spec' }` if none are provided